### PR TITLE
update to v6.4

### LINKED ISSUE #8: Request update package version
- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue: Please update `nano` to newer versions like [`v6.0`](https://www.nano-editor.org/download.php)

### PR DESCRIPTION
**Pull request by matthiasdiener**
Closes #8.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by matthiasdiener** [comment]
This is ready for review @conda-forge/nano

**Comment by matthiasdiener** [comment]
A gentle ping for a review @conda-forge/nano @conda-forge/core

**Comment by isuruf** [comment]
Looks like libiconv is not needed. See the warning.
```
WARNING (nano): run-exports library package conda-forge::libiconv-1.16-h516909a_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```

File: recipe/meta.yaml [hunk 1]
```diff
@@ -26,7 +25,6 @@ requirements:
     - libiconv
```
Comment (isuruf) [inline recipe/meta.yaml]: Remove this too

Comment (matthiasdiener) [reply recipe/meta.yaml]: Thanks, removed.

**Pull request merged**